src/components/ProhibitionIcon/ProhibitionIcon.js

```diff
@@ -42,4 +60,7 @@ export const ProhibitionIcon = ({ ariaLabel }) => {
   return renderIcon();
 };
 
-ProhibitionIcon.propTypes = { ariaLabel: PropTypes.string };
+ProhibitionIcon.propTypes = {
+  ariaLabel: PropTypes.string,
+  fieldName: PropTypes.string,
```

Comment:
Looks like this prop is required, isn't it?

Reply:
Done